README.rst

```diff
@@ -36,3 +40,93 @@ Non-openstack nodes will ignore the reason, and just do a regular reboot.
 If you don't specifiy the image, it will default to doing a rebuild with
 the existing image. If you don't have "rebuild" at the start of your
 reason, openstack nodes will do a regular reboot.
+
+slurm-stats
+^^^^^^^^^^^
+
+Command-line tool which transforms sacct output into a form that is more
+amenable for importing into elasticsearch/loki.
+
+It uses slurm's sacct to extract job stats of finished jobs. It stores the
+last timestamp in a file called "lasttimestamp". This means the tool can fetch all
+jobs since the last time stamp. If there is no stored timestime it defaults to
+fetching all jobs since midnight.
+
+Note: the default OpenHPC install doesn't enable the accounting service. It appears
```

Comment:
I'm not sure this is quite right - it does enable accounting but with `AccountingStorageType` defaulted to ` accounting_storage/filetxt` and it appears `accounting_storage/slurmdbd` is required for this.